crates/sui-move/src/disassemble.rs

```diff
@@ -57,6 +57,7 @@ impl Disassemble {
         let mut bytes = Vec::new();
         let mut file = BufReader::new(File::open(self.module_path)?);
         file.read_to_end(&mut bytes)?;
+        // this deserialized a module to the max version of the bytecode but it's OK here
```

Comment:
```suggestion
        // this deserialized a module to the max version of the bytecode but it's OK here
        // because it's not run as part of the deterministic replicated state machine.
```